src/create.rs

```diff
@@ -450,12 +465,17 @@ impl Execution {
         }
         compiler
             .arg(format!("-dVersion={}", version))
-            .arg(format!("-dPlatform={}", platform))
+            .arg(format!("-dPlatform={}", platform.wix_arch()?))
```

Comment:
Problem is, the warn will show up for everyone all the time, not just to users of the Platform variable.

We could grep for `var.Platform` in the provided WXS as a workaround, and show the WARN only if it's found there?

Reply:
Is grep available on Windows? Could the XPath crate used in the integration tests to search usage?

I briefly looked into this after commenting and had the same conclusion/thought. While a WARN statement would possibly be a nice active notice of deprecation, I am just not sure there is a good way to do without annoying every user. 

This does highlight that a general deprecation mechanism/process is probably needed. How do other Cargo subcommands handle this? We need to be able to make changes and evolve without fear while minimizing technical debt but without constantly breaking things for everyone. The goal of a tool is to make less work for everyone. Maybe when a command is executed, like the _create_ (default) command, WARN statements with all deprecations are printed, but we add a `--surpress-deprecations` flag? This would hide the notices, or it could work in the reverse (`--show-deprecations`) and be silent by default? Hmm, maybe this could be tied to the `--nocapture` flag, such that if the output is _not_ captured, the deprecation notices are printed along with the output from the WiX applications? 

I created #123 to discuss and track this idea.

For now, the course of action will be to document everything and give users some time to migrate. I created #121 to explicitly track this and provide a more visible indication of when it is removed.

Reply:
> Is grep available on Windows?

Well, by grepping, I meant looking for the string in the file, not literally invoking grep haha. XPath could certainly be used, I'll look into it.

Reply:
I think this would best be resolved in a separate PR, tracked by #121. I have a potential idea of how to do this very simply, but I'll need to read a bit on how WiX works with its preprocessor first, and it should be reviewed on its own.